eng/pipelines/templates/stages/archetype-sdk-bom.yml

```diff
@@ -2,6 +2,9 @@ parameters:
   Artifacts: []
   SDKType: bom
   ServiceDirectory: not-specified
+  Skip:
+    UpdatePackageVersion: true
```

Comment:
This should be:
```
Skip.UpdatePackageVersion: true
Skip.VersionVerification: true
```

Reply:
I was wondering what the syntax on this was 😃

Reply:
Updated to use `Skip.UpdatePackageVersion` and `Skip.VersionVerification`.